snippets/dart/account.code-snippets

```diff
@@ -0,0 +1,223 @@
+{
+    "Initialize account": {
+      "scope": "dart",
+      "prefix": ["account"],
```

Comment:
Prefix all the server snippets with `s` please. This would prevent for conflicting between the client and server SDKs